02-express-tutorial/answers/authorize.js

```diff
@@ -0,0 +1,13 @@
+const authorize = (req, res, next) => {
+    const {user} = req.query;
+    if (user === 'john'){
+        req.user = {name: 'john', id:3}
+        next()
+    }
+    else{
+        res.status(401).send('Unauthorized')
+    }
+    next()
```

Comment:
I  get an error when making a request that isn't authorized, because you are sending a response but not exiting the request (i.e. `return res.status(401).send('Unauthorized')`. 

Whenever you send a response, you should stop processing the request.

Reply:
Ya - that's bad example code. 🔨